blog/articles/event-drive-app-knative-eventing-kogito.md

```diff
@@ -0,0 +1,130 @@
+---
+Title: 'Orchestrating Events with Knative and Kogito'
+Author: Ricardo Zanini and Tihomir Surdilovic
+Author handle: https://twitter.com/zaninirica, https://twitter.com/tsurdilo
+Date: ''
+Description: A brief introduction of Knative Eventing integration with Kogito implementation of the Serverless Workflow specification
+Folder with media files: 'images/kogito'
+Blog URL: ''
+Labels: Demo
+Reviewers: ''
+
+---
+| Reviewer  | Date | Approval |
+| ------------- | ------------- | ------------- |
+| <!-- Your Github handle here -->  |   |:+1:, :monocle_face:, :-1:  |
+| <!-- Your Github handle here --> |   |  |
+
+[Kogito](https://kogito.kie.org/) is a platform for the development of cloud-native business automation applications. Kogito is designed targeting cloud-native architectures, and it comes with a series of features to make it easy for architects and developers to create business applications.
+
+Kogito implements the [CNCF Serverless Workflow Project](https://github.com/serverlessworkflow/specification), which is a specification for defining workflow models that orchestrate event-driven, serverless applications. It focuses on defining a vendor-neutral, platform-independent, and declarative workflow model for orchestrating services that can be used across multiple cloud and container platforms. To date, the Serverless Workflow specification is a CNCF sandbox project.
+
+As part of the Serverless Workflow implementation, Kogito offers a Kubernetes Operator to deploy these workflows with Knative. The goal is to make it as simple as possible to deploy and manage user-defined workflows in  cloud environments. Knative Eventing plays a very important role in this scenario by providing the underlying infrastructure for event-driven architectures.
+
+### Kogito Serverless Workflow
+
+To demonstrate how the Kogito workflow implementation works on event-driven architectures, we will use the [patient onboarding example](https://github.com/kiegroup/kogito-examples/tree/master/serverless-workflow-functions-events-quarkus). In this example, we simulate a workflow used in hospitals to onboard new patients and assign them to the correct doctor.
+
+The following image taken from the [specification examples page](https://github.com/serverlessworkflow/specification/blob/master/examples/examples.md#New-Patient-Onboarding) illustrates this workflow:
+
+![Patient Onboarding workflow representation](https://raw.githubusercontent.com/serverlessworkflow/specification/master/media/examples/example-patientonboarding.png)*Patient Onboarding workflow representation*
```

Comment:
```suggestion
![Patient onboarding workflow representation](https://raw.githubusercontent.com/serverlessworkflow/specification/master/media/examples/example-patientonboarding.png)*Patient Onboarding workflow representation*
```

You can just make the changes yourself instead of accepting the suggestion if it causes you to need to rebase etc, but I'll add them as suggestions anyway so the change is highlighted.